pom.xml

```diff
@@ -0,0 +1,154 @@
+                <artifactId>worldedit-bukkit</artifactId>
+                <version>6.1.4-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+                <groupId>com.sk89q</groupId>
+                <artifactId>worldguard</artifactId>
+                <version>6.1.1-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.8.2</version>
+        </dependency>
+        <dependency>
+            <groupId>net.sacredlabyrinth.Phaed</groupId>
+            <artifactId>PreciousStones</artifactId>
+            <version>10.1.2</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.sacredlabyrinth.phaed.simpleclans</groupId>
+            <artifactId>SimpleClans</artifactId>
+            <version>2.6.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.TechFortress</groupId>
+            <artifactId>GriefPrevention</artifactId>
+            <version>16.7.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.wasteofplastic</groupId>
+            <artifactId>askyblock</artifactId>
+            <version>3.0.6.9</version>
+        </dependency>
+        <dependency>
+            <groupId>com.griefcraft</groupId>
+            <artifactId>lwc</artifactId>
+            <version>4.6.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/LWC.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.massivecraft</groupId>
+            <artifactId>massivecore</artifactId>
+            <version>2.13.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/MassiveCore.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.massivecraft</groupId>
+            <artifactId>Factions</artifactId>
+            <version>2.13.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/Factions.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.palmergames</groupId>
+            <artifactId>Towny</artifactId>
+            <version>0.91.4.9</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/Towny.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.yi.acru.bukkit.Lockette</groupId>
+            <artifactId>Lockette</artifactId>
+            <version>0.91.4.9</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/Lockette.jar</systemPath>
```

Comment:
I looked into the licenses of the jars included in this PR regarding distribution, and they all seem to be okay:

- MassiveCore + Factions are [MIT](https://github.com/MassiveCraft/MassiveCore/issues/258#issuecomment-212266393) (although it seems like MassiveCore has finally been mavenized once again, so I'll see if I can find their maven repo and/or get it working with jitpack.)
- Towny is [Creative Commons (Attribution-NonCommercial-NoDerivs 3.0 Unported)](https://github.com/ElgarL/Towny/blob/master/README.txt)
- LWC just requires [this file](https://github.com/Hidendra/LWC/blob/master/LICENSE) to be included.
- Lockette uses [NPOSL-3.0](https://github.com/Acru/Lockette/blob/master/README.txt)

I did try using jitpack for some of these; LWC and Lockette had missing dependencies in their pom, and thus cannot build. Towny I've heard is a dependency nightmare, and I assumed MassiveCore + Factions weren't mavenized (they were originally, then de-mavenized due to not wanting others to "easily compile it for themselves because he gets rewards from bukkitdev downloads", then I guess they decided to mavenize again.)

If you still really don't want to include these jars, I can fork a couple of these and fix the dependency issues (and use jitpack), or just leave as-is and developers can download the jars themselves into the lib folder.